compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -1133,6 +1133,9 @@ public void visit(BLangValueType valueType) {
     @Override
     public void visit(BLangArrayType arrayType) {
         lookupNode(arrayType.elemtype);
+        for (BLangExpression size : arrayType.sizes) {
+            lookupNode(size);
+        }
```

Comment:
Can use `lookupNodes(arrayType.sizes)`

Reply:
Please note that the `arrayType.sizes` is of type `BLangExpression[]`.
The `lookupNodes()` accepts a parameter in a type of `List<? extends BLangNode> nodes`.  Therefore, a type casting was performed using `Arrays.asList()`.